lib/commons/rscommons/vector_ops.py

```diff
@@ -203,7 +203,11 @@ def copy_feature_class(in_layer_path: str, out_layer_path: str,
                 progbar.erase()  # get around the progressbar
                 log.warning('Feature with FID={} has no geometry. Skipping'.format(feature.GetFID()))
                 continue
-
+            if geom.GetGeometryType() == 2:
```

Comment:
2 little problems here:

1. Try to use the ogr enums so future you will know what he meant *(I couldn't tell you what geom type "2" is)* 
2. It's not *just* type "2", it's every linear type. [I've already abstracted these](https://github.com/Riverscapes/riverscapes-tools/blob/master/lib/commons/rscommons/classes/vector_base.py#L46) to a lookup list you can use: 

try something like
``` python
from vector_base import VectorBase
if geom.GetGeometryType() in VectorBase.LINE_TYPES:
   # do whatever
```

Reply:
Good advice. Changes made and pushed.